src/components/Navigation.js

```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+import { NavLink } from 'react-router-dom';
+import './main.css';
+
+const Navigation = () => {
+  return (
+    <div className="navlinks">
+      <NavLink activeClassName="selected" className="list" to="/" exact>
```

Comment:
Good choice🙌 going with navlink over link as it comes with some cool additional styling attributes like the one you're using here _activeClassName_